README.md

```diff
@@ -12,43 +10,125 @@ This package allows Python developers to write software that makes use of the We
 
 [Welkin Health API Documentation](https://developers.welkinhealth.com/)
 
-### Installing
+## Installing
 
 ```
 pip install welkin
 ```
 
-### Quick Start
+## Quick Start
 
 ```python
 from welkin import Client
 
-welkin = Client(
+client = Client(
     tenant="gh",
     instance="sb-demo",
     api_client="VBOPNRYRWJIP",
     secret_key="+}B{KGTG6#zG%P;tQm0C",
 )
+patient = client.Patient(
+  firstName="Foo",
+  lastName="Bar",
+).create()
+```
+
+## Examples
+
+### Create a Patient
+
+```python
+patient = client.Patient(
+  firstName="Peter",
+  lastName="Parker"
+).create()
+```
 
+### Get and Update an existing Patient
 
-### Patient methods
-patient = welkin.Patient(firstName="Foo", lastName="Bar").create()  # Create
+```python
+# single patient
+patient = client.Patient(
+  id=patient_id
```

Comment:
This change no longer illustrates the data type or format that is supposed to be used for patient IDs, and also no longer conforms to Black styling.